### ISSUE TITLE
RazorpayCheckout.open not calling success and failure callbacks

### ISSUE THREAD

**Issue by genioapps** [razorpay/react-native-razorpay#322]
## Description
For Android Integration :
The App is getting crashed on successful payment or failure payment.
The App is also not able to handle back and the close button on the Razorpay popup.
I have tested with the test environment, What I found the SDK is not calling its success or failure callback which we provide.

## Razorpay Package Version :
`react-native-razorpay` 2.2.4. 

## Java and Gradle Version (android)
Java 8 and Gradle 5.5.

### What you did:
Initiating the Payment in my react-native app using the below code snippet:
var options = {
		description: 'MY_BRAND_NAME',
		image: 'MY_IMAGE_URL',
		currency: 'INR',
		key: 'MY_TEST_KEY',
		amount: this.state.baseamt,
		name: ''MY_BRAND_NAME',
		order_id: order.payload.response.rzp_order,
		prefill: {
			email: this.state.email,
			contact: this.state.telephone,
			name: this.state.firstname,
		},
		theme: { color: '#6E175E' },
		
	};

`RazorpayCheckout.open(options)
    .then(data => {
	    // handle success
	    // The success callback is not getting called
	    console.log('razorpay', data);
	    this.RazorPay(
		    data.razorpay_payment_id,
		    data.razorpay_signature,
		    data.razorpay_order_id,
	    );
	    this.props.navigation.navigate('ThankYou', {
		    order_id: place_order.payload.response,
		    amount: this.state.baseamt,
		    paymenttype : this.state.paymentmode,
	    });
    })
    .catch(error => {
	    // handle failure
	    //console.log('razorpay error',  );
	    // THis failure callback is also not getting called.
	    this.props.navigation.navigate('ThankYou', {
		    order_id: 'place_order.payload.response',
		    amount: 'this.state.baseamt',
		    paymenttype : 'this.state.paymentmode',
	    });
	    ToastAndroid.showWithGravityAndOffset(
		    error.description,
		    ToastAndroid.LONG,
		    ToastAndroid.BOTTOM,
		    0,
		    10,
	    );
    });`

**Comment by sumedht** [comment]
@genioapps Will look into this.

**Comment by 0x61x** [comment]
@genioapps how did you solve this? I'm facing the same issue.

**Comment by genioapps** [comment]
> @genioapps how did you solve this? I'm facing the same issue.

Some other third party plugin was conflicting with the react-native-razorpay package on the native side.
The steps I have taken in order to resolve that are as mentioned below:

- Disabling all other react-native packages and test only Razorpay extension on the very first screen.
- After getting that it's working fine, I started enabling other extensions one by one, that's how I reached the extension which was causing the same.
- Then connected to that extension team and told them about the issue, they have fixed and updated the package on npm.

Thanks
